bin/citgm-all.js

```diff
@@ -56,7 +65,9 @@ var options = {
   level: app.verbose,
   npmLevel: app.npmLoglevel,
   timeoutLength: app.timeout,
-  tmpDir: app.tmpDir
+  tmpDir: app.tmpDir,
+  withTags: app.withTags,
+  excludeTags: app.excludeTags
```

Comment:
Okay, then you could do it in the line above right?

```js
  if (app.withTags) app.withTags.split(',');
  if (app.excludeTags) app.excludeTags.split(',');
```

You might have to use braces and more lines to appease the linter.